test/models/ui/comment_details_test.dart

```diff
@@ -47,5 +48,12 @@ void main() {
 
       expect(commentPost != commentPostOther, true);
     });
+
+    test("inequality test on owner ID", () {
+      final commentPost = unequalComments[0];
+      final commentPostOther = unequalComments[4];
+
+      expect(commentPost != commentPostOther, true);
```

Comment:
Very small suggestion, I found out that we could test inequality with the matchers `isNot(equals(***))`.
This would gives:
```suggestion
      expect(commentPost, isNot(equals(commentPostOther)));
```

I personnaly feels like it improves readability a little bit.
But this is just my point of view and I'd totally understand if you choose to stick with your approach :)

Reply:
You are completely right.

Changed in [cefc851](https://github.com/ProximaEPFL/proxima/pull/348/commits/cefc85186f44b65ca6bdfe9f097aaea631128868)

I also added this matcher to the other tests in this file :)

Thank you !

Reply:
Perfect :ok_hand: 
Thank you !